docs/reference/get_secrets_from_run.html

```diff
@@ -16,15 +16,13 @@
 <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>
```

Comment:
just FYI pkgdown docs update can be handled by build.  roxygen update is still needed

Reply:
sounds good.